src/pages/permissions/panels/GroupOrIdentityChangesTable.tsx

```diff
@@ -0,0 +1,185 @@
+import { Icon, Notification } from "@canonical/react-components";
+import { FC } from "react";
+import { IdentitiesOrGroupsChangeSummary } from "util/permissionIdentities";
+import Tag from "components/Tag";
+import { LxdIdentity } from "types/permissions";
+
+interface Props {
+  groupBy: "identity" | "group";
+  onChangeGroupBy: () => void;
+  loggedInIdentityID: string;
+  identityGroupsChangeSummary: IdentitiesOrGroupsChangeSummary;
+  groupIdentitiesChangeSummary: IdentitiesOrGroupsChangeSummary;
+  showLoggedInIdentityWarning: boolean;
+  identities: LxdIdentity[];
+}
+
+const generateRowsFromIdentityGroupChanges = (
+  identityGroupsChangeSummary: IdentitiesOrGroupsChangeSummary,
+  loggedInIdentityID: string,
+) => {
+  const identityIDs = Object.keys(identityGroupsChangeSummary);
+
+  const rows: JSX.Element[] = [];
+  for (const id of identityIDs) {
+    const groupChangesForIdentity = identityGroupsChangeSummary[id];
+    const identityLoggedIn = id === loggedInIdentityID;
```

Comment:
This is not the right check.

Scenario:
- user A and B
- logged in as user B
- A is in group foo, B is not
- On identities list select A and B, modify groups
- unselect group foo
- in the modal user A is shown with the "You" tag, even though B is the logged in user.


![image](https://github.com/canonical/lxd-ui/assets/1155472/c0001cbe-f302-470a-a174-ebcd32248d30)

Reply:
The same issue exists when coming from the groups list as below.
![image](https://github.com/canonical/lxd-ui/assets/1155472/2feee6fb-6fbb-44b9-93b9-883aa51d3756)

When I switch group by method, it is correct though
![image](https://github.com/canonical/lxd-ui/assets/1155472/41623dd6-d237-4266-becf-e72f2f3a14fc)

Reply:
sorry about this, if you look at line 57, I had the tag set to always visible. I was testing the styling for the tag earlier and seems like I didn't revert the change.